internal/toolname/toolname.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package toolname
+
+// ToolName of the CLI.
+var ToolName string
```

Comment:
why the new package/var? you could've kept the `config.ToolName` changing it to `var ToolName = "mongocli"` that way it keeps it's original behaviour and it can be changed at build time

Reply:
I have used "version" as an example for this but I see how your suggestion makes sense here. I will update it. Thanks!

Reply:
Updated